infra/mock_api/template/ecs.tf

```diff
@@ -109,21 +106,132 @@ resource "aws_ecs_task_definition" "mock-api-ecs-task-definition" {
         {
           containerPort : 8080
         }
-      ],
+      ]
+      readonlyRootFilesystem = true
+      linuxParameters = {
+        capabilities = {
+          drop = ["ALL"]
+        },
+        initProcessEnabled = true
+      }
+      environment : [
+        {
+          "name" : "POSTGRES_USER"
+          "value" : "${data.aws_ssm_parameter.db_username.value}"
+        },
+        {
+          "name" : "POSTGRES_PASSWORD"
+          "value" : "${data.aws_ssm_parameter.db_pw.value}"
+        },
+        {
+          "name" : "POSTGRES_DB"
+          "value" : "${data.aws_ssm_parameter.db.value}"
+        },
+        {
+          "name" : "DB_HOST"
+          "value" : "test-wic-mt.cfsgsglh2mrn.us-east-1.rds.amazonaws.com"
+        }
+      ]
       logConfiguration = {
         logDriver = "awslogs",
         options = {
-          "awslogs-group"  = "${data.aws_cloudwatch_log_group.mock_api}"
-          "awslogs-region" = "us-east-1"
+          "awslogs-group"         = "mock-api",
+          "awslogs-region"        = "us-east-1",
+          "awslogs-stream-prefix" = "mock-api"
         }
       }
-      readonlyRootFilesystem = true
+    }
+  ])
+}
+# ------------------------------------------------------------------------------
+#    ECS task to Handle CSVs
+#
+# ------------------------------------------------------------------------------
+# resource "aws_iam_role" "handle-csv" {
+#   name = "handle-csv-role"
+#   description = "allows an ECS task to generate CSVs and manage their storage"
+#   policy = data.aws_iam_policy_document.handle-csv
+# }
+# resource "aws_iam_role_policy" "handle-csv" {
+# add s3 perms (read, write, list)
+# read rds?
+# }
+
+resource "aws_security_group" "handle-csv" {
```

Comment:
Should we make this something more generic like `ecs-task-security-group`? That way we don't need to re-specify it for each subsequent ECS task (+ lets me hardcode pointing to it in run-task.sh).